static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -70,10 +72,49 @@
     }
 }
 
+.adverts--tone-supported {
+    background: none;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        border-top: 1px solid $news-main-2;
+
+        @include mq(tablet) {
+            left: calc(50% - 370px);
```

Comment:
Now that I come think of it, I can use this technique and remove half the properties in the media queries. Sweet 🏄